servicecatalog-client/pom.xml

```diff
@@ -0,0 +1,224 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Copyright (C) 2015 Red Hat, Inc.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+            http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <parent>
+    <artifactId>kubernetes-client-project</artifactId>
+    <groupId>io.fabric8</groupId>
+    <version>4.3-SNAPSHOT</version>
+  </parent>
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>me.snowdrop</groupId>
+  <artifactId>servicecatalog-client</artifactId>
+  <name>Fabric8 :: Service Catalog :: Client</name>
+  <packaging>bundle</packaging>
+
+  <properties>
+    <useIncrementalCompilation>false</useIncrementalCompilation>
+    <osgi.import>
+      io.fabric8.kubernetes.api.builder,
+      !me.snowdrop.servicecatalog.api.client.*,
+      *
+    </osgi.import>
+    <osgi.export>
+      me.snowdrop.servicecatalog.api.client.*
+    </osgi.export>
+
+    <jsonassert.version>1.2.3</jsonassert.version>
+    <lombok.version>1.16.4</lombok.version>
+    <validation-api.version>1.1.0.Final</validation-api.version>
+  </properties>
+
+  <dependencies>
+    <dependency>
+      <groupId>me.snowdrop</groupId>
+      <artifactId>servicecatalog-model</artifactId>
+      <version>${svc.version}</version>
+    </dependency>
+
+    <dependency>
+      <groupId>io.fabric8</groupId>
+      <artifactId>kubernetes-client</artifactId>
+      <version>4.2.2</version>
+      <exclusions>
```

Comment:
${project.version} instead of 4.2.2 (now that it's part of the repo)

Reply:
Fixed

Reply:
Fixed